README.md

```diff
@@ -252,11 +252,10 @@ An overview of the flow:
 
 ## Docker images
 
-Pre-release docker image is available at
+Docker images are available [here](https://console.cloud.google.com/gcr/images/gardener-project/EU/gardener/oidc-webhook-authenticator) or you can choose to pull the latest pre-release version with the following command:
```

Comment:
If you open eu.gcr.io/gardener-project/gardener/oidc-webhook-authenticator in a browser, it will forward you to the same page.